pkg/health/health_pod.go

```diff
@@ -194,6 +216,16 @@ func getCorev1PodHealth(pod *corev1.Pod) (*HealthStatus, error) {
 					}, nil
 				}
 			}
+
+			// Pod isn't ready but all containers are
+			if allContainersReady {
+				return &HealthStatus{
```

Comment:
Need to add handle a pod that is not ready, but is still within its allowed sstartup window - > Unknown health, Status => Starting, Message: Waiting for readiness probe